beacon-chain/p2p/discovery.go

```diff
@@ -99,13 +99,13 @@ func (s *Service) createListener(
 ) (*discover.UDPv5, error) {
 	// Listen to all network interfaces
 	// for both ip protocols.
-	var networkVersion string
+	networkVersion := "udp"
```

Comment:
Maybe define `networkVersion` next to its usage before L129? 

Otherwise it's a little hard to tell what is going on : )

Reply:
Alright done